src/ui-test/draft-manifest-files.test.ts

```diff
@@ -0,0 +1,367 @@
+   const pathToWorkspace =
+      path.resolve(__dirname, '../../src/ui-test/test-repo/flask-hello-world') +
+      '/';
+   let browser: VSBrowser;
+
+   before(async function () {
+      this.timeout(100000);
+      browser = VSBrowser.instance;
+      await setWorkspace(pathToWorkspace, browser);
+   });
+
+   it('drafts manifest files', async function () {
+      this.timeout(200000);
+      //open command palette
+      let prompt = await new Workbench().openCommandPrompt();
+      const picks = await prompt.getQuickPicks();
+      assert.notStrictEqual(picks.length, 0);
+
+      //select quick pick to draft manifest files
+      await prompt.selectQuickPick(
+         'AKS Developer: Draft a Kubernetes Deployment and Service'
+      );
+      await browser.driver.wait(
+         until.elementIsVisible(
+            await browser.driver.findElement(
+               By.xpath(".//input[@aria-describedby='quickInput_message']")
+            )
+         )
+      );
+      await prompt.confirm();
+      await browser.driver.wait(
+         until.elementLocated(By.className('quick-input-list'))
+      );
+      await prompt.selectQuickPick('Manifests');
+      await browser.driver.wait(
+         until.elementLocated(
+            By.xpath(".//input[@aria-describedby='quickInput_message']")
+         )
+      );
+      await prompt.setText('flask-hello-world');
+      await prompt.confirm();
+      await browser.driver.wait(
+         until.elementTextContains(
+            await browser.driver.findElement(By.css('div.quick-input-message')),
+            'Port'
+         )
+      );
+      await browser.driver.wait(
+         until.elementLocated(
+            By.xpath(".//input[@aria-describedby='quickInput_message']")
+         )
+      );
+      await prompt.setText('8080');
+      await browser.driver.sleep(3000);
+
+      await prompt.confirm();
+
+      await browser.driver.wait(
+         until.elementLocated(By.className('quick-input-list'))
+      );
+      await prompt.selectQuickPick('default');
+      await browser.driver.wait(
+         until.elementLocated(By.className('quick-input-list'))
+      );
+      await prompt.selectQuickPick('Other');
+      await browser.driver.wait(
+         until.elementIsVisible(
+            await browser.driver.findElement(
+               By.xpath(".//input[@aria-describedby='quickInput_message']")
+            )
+         )
+      );
+      await prompt.setText('test-image');
+      await browser.driver.wait(
+         until.elementIsVisible(
+            await browser.driver.findElement(
+               By.xpath(".//input[@aria-describedby='quickInput_message']")
+            )
+         )
+      );
+      await prompt.confirm();
+      await browser.driver.wait(
+         until.elementTextContains(
+            await browser.driver.findElement(By.css('div.quick-input-message')),
+            'Image Tag'
+         )
+      );
+      await prompt.confirm();
+      await browser.driver.sleep(3000);
```

Comment:
This wait is for the completion of the task to run and generate files correct?

Reply:
Yes, that's right!

Reply:
We should be able to wait for an element like the others. We should be able to make a custom watcher that waits for the files.

It's a requirement that this command opens the files in the editor so we should be able to wait for that.

Reply:
Maybe theres something we could tap into besides this? I think after every run we always have some sort of popup confirming the run or displaying the error that occurred. could we try to wait for that?

Reply:
I've added locators for the generated files opened in the editor, as well as added an assertion for the correct notification after each run.